src/vorta/assets/UI/sourcetab.ui

```diff
@@ -13,140 +13,141 @@
-        <property name="text">
-         <string>File Count</string>
+        <property name="selectionMode">
+         <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
-       </column>
-      </widget>
-     </item>
-     <item>
-      <layout class="QVBoxLayout" name="verticalLayout">
-       <item>
-        <widget class="QPushButton" name="sourceAddFolder">
-         <property name="text">
-          <string>Add Folder</string>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QPushButton" name="sourceAddFile">
-         <property name="text">
-          <string>Add File(s)</string>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QPushButton" name="sourceRemove">
+        <property name="selectionBehavior">
+         <enum>QAbstractItemView::SelectRows</enum>
+        </property>
+        <property name="showGrid">
+         <bool>false</bool>
+        </property>
+        <attribute name="horizontalHeaderVisible">
+         <bool>false</bool>
+        </attribute>
+        <attribute name="horizontalHeaderHighlightSections">
+         <bool>false</bool>
+        </attribute>
+        <attribute name="verticalHeaderVisible">
+         <bool>false</bool>
+        </attribute>
+        <attribute name="verticalHeaderHighlightSections">
+         <bool>false</bool>
+        </attribute>
+        <column>
          <property name="text">
-          <string>Remove</string>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QPushButton" name="paste">
-         <property name="toolTip">
-          <string>One folder or file per line</string>
+          <string>Path</string>
          </property>
+        </column>
+        <column>
          <property name="text">
-          <string>Paste Folders/Files</string>
+          <string>Size</string>
          </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QPushButton" name="sourcesUpdate">
+        </column>
+        <column>
          <property name="text">
-          <string>Recalculate sizes</string>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <spacer name="verticalSpacer">
-         <property name="orientation">
-          <enum>Qt::Vertical</enum>
+          <string>File Count</string>
          </property>
-         <property name="sizeHint" stdset="0">
-          <size>
-           <width>20</width>
-           <height>40</height>
-          </size>
-         </property>
-        </spacer>
-       </item>
-      </layout>
-     </item>
-    </layout>
+        </column>
+       </widget>
+      </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QPushButton" name="updateButton">
+          <property name="toolTip">
+           <string>Recalculate source size and file count</string>
```

Comment:
It seems like Qt on MacOS doesn't support `QToolbuttons` but only `QPushbuttons`.

Reply:
Or it might relate to this note in the Qt docs:

> On macOS when a push button's width becomes smaller than 50 or its height becomes smaller than 30, the button's corners are changed from round to square. Use the setMinimumSize() function to prevent this behavior.